resources/schema.gql

```diff
@@ -3383,7 +3383,12 @@ type MoleculeDataRoomMutV2 {
 	"""
 	Updates the metadata of a file in the data room.
 	"""
-	updateFileMetadata(ref: DatasetID!, accessLevel: String!, changeBy: String!, description: String, categories: [String!], tags: [String!], contentText: String, encryptionMetadata: MoleculeEncryptionMetadataInput): MoleculeDataRoomUpdateFileMetadataResult!
+	updateFileMetadata(		ref: DatasetID!,
```

Comment:
Either add comments too all arguments, or remove your's, as this made schema unreadable